src/mcts/params.h

```diff
@@ -99,6 +99,10 @@ class SearchParams {
     return options_.Get<std::string>(kScoreTypeId.GetId());
   }
   FillEmptyHistory GetHistoryFill() const { return kHistoryFill; }
+  int GetSidetomoveDrawScore() const { return kSidetomoveDrawScore; }
+  int GetOpponentDrawScore() const { return kOpponentDrawScore; }
+  int GetWhiteDrawDelta() const { return kWhiteDrawDelta; }
+  int GetBlackDrawDelta() const { return kBlackDrawDelta; }
```

Comment:
These seem to want to be `float`s not `int`s as you divide the provided value by `100.0f`. But why not just make the params from -1.0f to 1.0f instead of integers?

Separately, not sure if it's worthwhile to keep the internal names consistent with the option. E.g., `GetBlackDrawDelta` vs `DrawScoreBlack`

Reply:
oops, good catch! That's why there was no performance degradation for small values, I'll retest after the fix.
Params are integers because in UCI integers are handled much nicer (we use uci string type for float).

For variable names, probably makes sense to rename them, that's a leftover from the first variant.